packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -619,11 +619,11 @@ export class GarbageCollector implements IGarbageCollector {
                 ["/"],
                 logger,
             );
-            const gcStats = this.generateStatsAndLogEvents(gcResult);
+            const gcStats = this.generateStatsAndLogEvents(gcResult, logger);
```

Comment:
This should be two different methods. `generateStats` and `logGCEvents`.

Reply:
This PR is just ensuring that the summary logger is used for GC events during summarize so that they can be correlated. This refactor should be discussed separately.